build-deb.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+
+# Build a Jellyfin .deb file with Docker on Linux
+# Places the output .deb file in the parent directory
+
+set -o xtrace
+set -o errexit
+set -o nounset
+
+package_temporary_dir="`mktemp -d`"
+current_user="`whoami`"
+image_name="jellyfin-debuild"
+
+cleanup() {
```

Comment:
Add `set +e` here. Otherwise, an error in the `docker image` step will result in the next command never executing.

Reply:
Remove errexit entirely since we clean up in a trap anyways.

Reply:
Wait no - keep `set -e` for the whole script, but disable it in the cleanup.